Directory.Build.targets

```diff
@@ -17,19 +18,4 @@
   <PropertyGroup>
     <Version Condition="'$(IsStableRelease)' == 'true' and '$(IsShipping)' == 'true'">$(VersionPrefix)</Version>
```

Comment:
I would expect both sourcelink and version info to be all calculated by arcade instead of this repo, so idieally you should be able to delete all this.

Reply:
I think this is what overrides the version number when we set the `IsStableRelease` variable though.